app/src/components/domain/RiskImminentEvents/Pdc/index.tsx

```diff
@@ -20,6 +39,32 @@ import {
 import EventDetails from './EventDetails';
 import EventListItem from './EventListItem';
 
+const WARNING_ALERT_TYPE = 'WARNING';
+const WATCH_ALERT_TYPE = 'WATCH';
+const ADVISORY_ALERT_TYPE = 'ADVISORY';
+const INFORMATION_ALERT_TYPE = 'INFORMATION';
+
+type AlertType = typeof WARNING_ALERT_TYPE
+    | typeof WATCH_ALERT_TYPE
+    | typeof ADVISORY_ALERT_TYPE
+    | typeof INFORMATION_ALERT_TYPE;
+
+function getAlertType(alertType: AlertType) {
+    if (alertType === WARNING_ALERT_TYPE) {
+        return CYCLONE_RED_ALERT_LEVEL;
+    }
+    if (alertType === WATCH_ALERT_TYPE) {
+        return CYCLONE_ORANGE_ALERT_LEVEL;
+    }
+    if (alertType === ADVISORY_ALERT_TYPE) {
+        return CYCLONE_GREEN_ALERT_LEVEL;
+    }
+    if (alertType === INFORMATION_ALERT_TYPE) {
+        return CYCLONE_BLUE_ALERT_LEVEL;
+    }
+    return undefined;
```

Comment:
A similar function for Gdacs was returning `none`. Let's make this consistent and use a consistent format.